torch/distributed/nn/api/remote_module.py

```diff
@@ -155,18 +155,22 @@ def __init__(
         args = args if args is not None else ()
         kwargs = kwargs if kwargs is not None else {}
 
-        self.on = on
+        [self.on, self.device] = remote_device.split("/")
```

Comment:
If this split is invalid, the user would get an error like "too many values to unpack". Ideally, we want to raise an error like this to the user if the format is not valid: "Couldn't parse remote_device: {}, the accepted format is <workername>/<device>".format(remote_device)". This makes it much easier for the user to understand what went wrong.

In addition to this, specifying just "workername" should be allowed as well. In this case, we assume the default is "cpu".